x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -100,29 +100,39 @@ const defaultPostPushToService = {
   pushCaseToExternalService,
 };
 
-const fillFormReactTestingLib = async (renderResult: RenderResult) => {
+const sampleDataWithoutTags = {
+  ...sampleData,
+  tags: [],
+};
+
+const fillFormReactTestingLib = async (renderResult: RenderResult, withTags = false) => {
```

Comment:
super nit: I try not to have functions take booleans because it makes it difficult to know what the parameter means when you just look at the function call like this:

```
fillFormReactTestingLib(renderResult, true);
```

One option would be to pass an object so it's descriptive (I'm not sure how annoying that'd be to have to update throughout this whole file though)

```
const fillFormReactTestingLib = async ({renderResult, withTags = false}: {renderResult: RenderResult, withTags: boolean})
```

Or another option would be to create a new function that wraps this one with descriptive name indicating that it is filling the form with tags:

```
const fillFormIncludeTags = async () => {
   await fillFormReactTestingLib(renderResult, true);
}
```

Reply:
I was lazy, you are right!